DATA/common/setenv_calib.sh

```diff
@@ -15,79 +15,109 @@ SOURCE_GUARD_SETENV_CALIB=1
 # Remove this line to only send unbinned residuals
 if [[ -z "$CALIB_TPC_SCDCALIB_SENDTRKDATA" ]];  then export CALIB_TPC_SCDCALIB_SENDTRKDATA=1; fi
 
+# define the conditions for each calibration
+if has_detector_calib ITS && has_detectors_reco ITS && has_detector_matching PRIMVTX && [[ ! -z "$VERTEXING_SOURCES" ]]; then CAN_DO_CALIB_PRIMVTX_MEANVTX=1; else CAN_DO_CALIB_PRIMVTX_MEANVTX=1; fi
```

Comment:
is this a typo? shouldn't it be `then CAN_DO_CALIB_PRIMVTX_MEANVTX=1; else CAN_DO_CALIB_PRIMVTX_MEANVTX=0; fi`?

Reply:
Yes, thanks for noticing.